fgpyo/io/tests/test_io.py

```diff
@@ -0,0 +1,116 @@
+"""Basic tests for io module"""
+
+import io
```

Comment:
run isort on the imports here please

Reply:
@NatPRoach I ran `isort` directly on [fgpyo/io/tests/test_io.py](https://github.com/fulcrumgenomics/fgpyo/pull/30/files#diff-1d74bdaa98107b4715a13b022dd0b47df7fca024ff9f83861aab4718e072befa) and the format is changed back after running `./ci/check.sh`

Reply:
@NatPRoach I ran `isort` directly and the format was changed back after running `./ci/check.sh`